hist/hist/src/TGraph.cxx

```diff
@@ -2189,7 +2189,8 @@ void TGraph::SaveHistogramAndFunctions(std::ostream &out, const char *varname, I
       TString hname = fHistogram->GetName();
       fHistogram->SetName(TString::Format("Graph_%s%d", hname.Data(), frameNumber).Data());
       fHistogram->SavePrimitive(out, "nodraw");
-      out << "   "<<varname<<"->SetHistogram(" << fHistogram->GetName() << ");" << std::endl;
+      out << "   " <<varname << "->SetHistogram(" << gInterpreter->MapCppName(fHistogram->GetName()) << ");"
```

Comment:
Not sure if `gInterpreter->MapCppName` required here.

Histogram name is set as specified before - and only can be find via that name.

Reply:
I see now code of `TH1::SavePrimitive`. Yes, one should keep here `gInterpreter->MapCppName`